### DISCUSSION TITLE
Lesson 7 - Interactions.s.sol

### DISCUSSION THREAD

**Discussion by Ibby-debug** [Cyfrin/foundry-full-course-cu#970] [General]
I would like to know in the ```‎IntegrationsTest‎```, who the hell is funding the ```FundMe``` contract. 

It's ```setUp()``` does two things. 
1. Stores deployed contract in the fundMe interface using the ```deployFundMe``` script. 
2. Deals ```USER``` with a ```STARTING_BALANCE```. 

Then, we initialize a fundFundMe script, call its ```fundFundMe()``` function passing in address of FundMe contract as a parameter. 

Is it the fundFundMe script the funder ? 

Thanks

**Comment by 0xusmanf** [comment]
Its Foundry's default sender address `0x1804c8AB1F12E6bbf3894d4083f33e07309d1f38` and it will always be until or unless you explicitly specify your own deployer key in `vm.startBroadcast(deployerKey)` or in the terminal by `--private-key`.

**Reply by Ibby-debug** [reply]
this explanation makes since!
